README.md

```diff
@@ -101,6 +101,29 @@ from the new application's `apps.py` file.
    Valid values for `verbosity` are `"queryPlanner"` (default),
    `"executionStats"`, and `"allPlansExecution"`.
 
+## Why are we building a MongoDB-Django integration?
+
+Building a robust solution that makes Django and NoSQL databases work together well has historically been challenging due to Django's ORM being built around relational database principles. While there have been several attempts in the past to create NoSQL (and MongoDB) integrations for Django, many of these projects have not achieved lasting success for various reasons. So, why have we chosen to tackle this problem now?
+
+According to the 2023 Django Developer Survey, 8% of respondents reported using MongoDB as a database backend with Django. This figure has shown a consistent upward trend in recent years (see 2021 and 2022 surveys). Additionally, numerous community members and customers have expressed a strong interest in leveraging Django alongside MongoDB.
+Our goal is to meet the needs of the developer community by providing an exceptional experience when working with MongoDB. Therefore, we are embarking on this journey to create a reliable integration. **Please note that the current project is in a very early stage of development and is not yet ready for PoC or Production use cases.**
+
+## Current capabilities
+
+At this stage, the project should let you replicate the djangoproject.com tutorial with minimal changes. You will likely be able to run it passing a portion of the standard Django test suite but not all of it. Some key features that we have added so far are:
```

Comment:
This section is too vague to be useful. There are some parts of the list below that will work and some parts that won't. As I have said before, we should document what doesn't work.